### PR TITLE
Update pt-PT.json5

### PR DESCRIPTION
**Pull request by rsaraivac**
Addition of new translation keys (following DSpace 7.6.1 release) & revision/improvement of some of the already existing.

## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes #`issue-number` (if this fixes an issue ticket)
* Requires DSpace/DSpace#`pr-number` (if a REST API PR is required to test this)

## Description
Short summary of changes (1-2 sentences).

## Instructions for Reviewers
Please add a more detailed description of the changes made by your PR. At a minimum, providing a bulleted list of changes in your PR is helpful to reviewers.

List of changes in this PR:
* First, ...
* Second, ...

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [ ] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [ ] My PR passes [ESLint](https://eslint.org/) validation using `yarn lint`
- [ ] My PR doesn't introduce circular dependencies (verified via `yarn check-circ-deps`)
- [ ] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [ ] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new libraries/dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.
- [ ] If my PR includes new features or configurations, I've provided basic technical documentation in the PR itself.
- [ ] If my PR fixes an issue ticket, I've [linked them together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

### CODE REVIEW

File: src/assets/i18n/pt-PT.json5 [hunk 1]
```diff
@@ -215,6 +215,9 @@
   // "admin.registries.bitstream-formats.title": "Bitstream Format Registry",
   "admin.registries.bitstream-formats.title": "Registo de formatos de ficheiro",
 
+  // "admin.registries.bitstream-formats.select": "Select bitstream format",
+  "admin.registries.bitstream-formats.select": "Selecionar formato de 'bitstream'",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:219]: Can you consider to use "ficheiro" to be coeherent?

File: src/assets/i18n/pt-PT.json5 [hunk 2]
```diff
@@ -7703,45 +7870,109 @@
...
 
-  // "search.filters.namedresourcetype.Workflow": "Workflow",
-  "search.filters.namedresourcetype.Workflow": "Em fluxo de trabalho",
+  // "access-control-add-to-existing": "Add to existing ones",
+  "access-control-add-to-existing": "Adicionar aos existentes",
 
-  // "search.filters.namedresourcetype.Workspace": "Workspace",
-  "search.filters.namedresourcetype.Workspace": "Por terminar",
+  // "access-control-limit-to-specific": "Limit the changes to specific bitstreams",
+  "access-control-limit-to-specific": "Limitar as alterações a 'bitstreams' específicos",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7916]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 3]
```diff
@@ -7703,45 +7870,109 @@
...
+
+  // "access-control-process-all-bitstreams": "Update all the bitstreams in the item",
+  "access-control-process-all-bitstreams": "Atualizar todos os 'bitstreams' no item",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7919]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 4]
```diff
@@ -7703,45 +7870,109 @@
...
+
+  // "access-control-bitstreams-selected": "bitstreams selected",
+  "access-control-bitstreams-selected": "bitstreams selecionados",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7922]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 5]
```diff
@@ -7703,45 +7870,109 @@
...
+
+  // "access-control-bitstreams-select": "Select bitstreams",
+  "access-control-bitstreams-select": "Selecionar 'bitstreams'",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7925]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 6]
```diff
@@ -7703,45 +7870,109 @@
...
+  "access-control-execute": "Executar",
+
+  // "access-control-add-more": "Add more",
+  "access-control-add-more": "Adicionar mais",
+
+  // "access-control-remove": "Remove access condition",
+  "access-control-remove": "Remover condição de acesso",
+
+  // "access-control-select-bitstreams-modal.title": "Select bitstreams",
+  "access-control-select-bitstreams-modal.title": "Selecionar 'bitstreams'",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7940]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 7]
```diff
@@ -7703,45 +7870,109 @@
...
 
-  // "mydspace.status.mydspaceValidation": "Validation",
-  "mydspace.status.mydspaceValidation": "Em validação",
+  // "access-control-bitstream-header-toggle": "Bitstreams",
+  "access-control-bitstream-header-toggle": "Bitstreams",
 
-  // "mydspace.status.mydspaceWaitingController": "Waiting for Controller",
-  "mydspace.status.mydspaceWaitingController": "Aguarda validador",
+  // "access-control-bitstream-toggle.enable": "Enable option to perform changes on the bitstreams",
+  "access-control-bitstream-toggle.enable": "Ativar a opção para efetuar alterações nos 'bitstreams'",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7895]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 8]
```diff
@@ -7703,45 +7870,109 @@
...
 
-  // "mydspace.status.mydspaceWorkflow": "Workflow",
-  "mydspace.status.mydspaceWorkflow": "Em fluxo de trabalho",
+  // "access-control-bitstream-toggle.disable": "Disable option to perform changes on the bitstreams",
+  "access-control-bitstream-toggle.disable": "Desativar a opção para efetuar alterações nos 'bitstreams'",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7898]: Can you use "ficheiro" instead of "bitstream"?

File: src/assets/i18n/pt-PT.json5 [hunk 9]
```diff
@@ -7703,45 +7870,109 @@
   // "admin.system-wide-alert.title": "System-wide Alerts",
   "admin.system-wide-alert.title": "Alertas gerais do sistema",
 
+  // "item-access-control-title": "This form allows you to perform changes to the access conditions of the item's metadata or its bitstreams.",
+  "item-access-control-title": "Este formulário permite-lhe efetuar alterações às condições de acesso dos metadados do item ou dos seus 'bitstreams'.",
```
Comment (paulo-graca) [inline src/assets/i18n/pt-PT.json5:7874]: Can you use "ficheiro" instead of "bitstream"?

**Review by paulo-graca — COMMENTED** [review]
Thank you @rsaraivac for this contribution! I've just some comments, could you can check if they make sense to you?

**Comment by tdonohue** [comment]
@rsaraivac : If you'd like this to be considered for 8.0 (and backported to 7.x), [please address the feedback above](https://github.com/DSpace/dspace-angular/pull/2706#pullrequestreview-1841084334).  Otherwise, we'll need to reschedule for 9.0

**Comment by tdonohue** [comment]
@paulo-graca : Could you give this a re-review?  It looks like your feedback has been addressed.

**Review by paulo-graca — APPROVED** [review]
Thank you @rsaraivac!

**Pull request merged**